assembly/src/tokens/mod.rs

```diff
@@ -150,7 +150,7 @@ impl<'a> Token<'a> {
         let is_export = self.parts[0] == Self::EXPORT;
         let (name_str, num_locals) = match self.num_parts() {
             0 => unreachable!(),
-            1 => return Err(ParsingError::missing_param(self)),
+            1 => return Err(ParsingError::missing_param(self, "[proc|export].<procedure_name?>")),
```

Comment:
nit: We don't need `?` at the end of `procedure_name` as procedure name is not an optional parameter.

Reply:
Yes, true. Corrected this.